SpamPkg/Library/SmmCpuFeaturesLib/SmmStm.c

```diff
@@ -329,6 +340,7 @@ DiscoverSmiEntryInFvHobs (
         }
       } while (!EFI_ERROR (Status));
       Hob.Raw = GetNextHob (EFI_HOB_TYPE_FV, GET_NEXT_HOB (Hob));
+      Status = EFI_SUCCESS;
     }
   } while (Hob.Raw != NULL);
 

```

Comment:
Also, this line below, can we please check the found status for all 3 entities?

Reply:
See other comment.